### ISSUE TITLE
Plugin crashes on start when installed in PhpStorm 2019.1 on Debian Linux amd64

### ISSUE THREAD

**Issue by christopher-john-czettel** [kevinlinxp/intellij-sublime-snippets-support#3]
Hi!

When installing "Sublime Snippets Support: v1.2.5"-Plugin on PhpStorm 2019.1 I get following error message and the plugin is automatically disabled before PhpStorm restarts itself again.

Running on Debian buster (4.19.0-2-amd64) using i3wm as a normal user.

Not sure if this is related to the new release of 2019.1 or my local setup. 

Please let me know if I can help resolve this issue!

Regards,

Chris

```
2019-03-29 14:13:37,412 [   3910]  ERROR - llij.ide.plugins.PluginManager - com.intellij.ide.plugins.PluginManager$StartupAbortedException: Fatal error initializing plugin com.kevinlinxp.sublimeSnippetsSupport                                                                                                                                                           
java.lang.RuntimeException: com.intellij.ide.plugins.PluginManager$StartupAbortedException: Fatal error initializing plugin com.kevinlinxp.sublimeSnippetsSupport
        at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:215)
        at com.intellij.idea.IdeaApplication.lambda$initApplication$0(IdeaApplication.java:74)
        at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
        at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
        at java.awt.EventQueue.access$500(EventQueue.java:98)
        at java.awt.EventQueue$3.run(EventQueue.java:715)
        at java.awt.EventQueue$3.run(EventQueue.java:709)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
        at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
        at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:347)
        at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
        at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
        at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
        at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
        at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
        at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: com.intellij.ide.plugins.PluginManager$StartupAbortedException: Fatal error initializing plugin com.kevinlinxp.sublimeSnippetsSupport
        at com.intellij.ide.plugins.PluginManager.handleComponentError(PluginManager.java:253)
        at com.intellij.openapi.components.impl.PlatformComponentManagerImpl.handleInitComponentError(PlatformComponentManagerImpl.java:43)
        at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:500)
        at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponents(ComponentManagerImpl.java:104)
        at com.intellij.openapi.application.impl.ApplicationImpl.lambda$createComponents$9(ApplicationImpl.java:449)
        at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:164)
        at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:586)
        at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
        at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:86)
        at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:151)
        at com.intellij.openapi.application.impl.ApplicationImpl.createComponents(ApplicationImpl.java:473)
        at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:88)
        at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:407)
        at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:393)
        at com.intellij.idea.IdeaApplication.run(IdeaApplication.java:208)
        ... 16 more
Caused by: com.intellij.diagnostic.PluginException: java.nio.file.AccessDeniedException: RECOVERED_FILES [Plugin: com.kevinlinxp.sublimeSnippetsSupport]
        ... 31 more
Caused by: java.io.UncheckedIOException: java.nio.file.AccessDeniedException: RECOVERED_FILES
        at java.nio.file.FileTreeIterator.fetchNextIfNeeded(FileTreeIterator.java:88)
        at java.nio.file.FileTreeIterator.hasNext(FileTreeIterator.java:104)
        at java.util.Iterator.forEachRemaining(Iterator.java:115)
        at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
        at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499)
        at com.kevinlinxp.sublimeSnippetsSupport.SublimeSnippetsSupportSettings.reloadSublimeSnippetsAsLiveTemplates(SublimeSnippetsSupportSettings.kt:69)
        at com.kevinlinxp.sublimeSnippetsSupport.PluginRegistration.initComponent(PluginRegistration.kt:14)
        at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:482)
        ... 28 more
Caused by: java.nio.file.AccessDeniedException: RECOVERED_FILES
        at sun.nio.fs.UnixException.translateToIOException(UnixException.java:84)
        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
        at sun.nio.fs.UnixFileSystemProvider.newDirectoryStream(UnixFileSystemProvider.java:427)
        at java.nio.file.Files.newDirectoryStream(Files.java:457)
        at java.nio.file.FileTreeWalker.visit(FileTreeWalker.java:300)
        at java.nio.file.FileTreeWalker.next(FileTreeWalker.java:372)
        at java.nio.file.FileTreeIterator.fetchNextIfNeeded(FileTreeIterator.java:84)
        ... 39 more
2019-03-29 14:13:37,415 [   3913]  ERROR - llij.ide.plugins.PluginManager - PhpStorm 2019.1  Build #PS-191.6183.95
2019-03-29 14:13:37,415 [   3913]  ERROR - llij.ide.plugins.PluginManager - JDK: 1.8.0_202-release; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o
2019-03-29 14:13:37,415 [   3913]  ERROR - llij.ide.plugins.PluginManager - OS: Linux
2019-03-29 14:13:37,422 [   3920]  ERROR - llij.ide.plugins.PluginManager - Plugin to blame: Sublime Snippets Support version: 1.2.5
```

`

**Comment by kevinlinxp** [comment]
Hi @christopher-john-czettel,

Many thanks for creating this issue.

It seems to a bug caused by line 39 of [SublimeSnippetsSupportSettings.kt](https://github.com/kevinlinxp/intellij-sublime-snippets-support/blob/master/src/main/kotlin/com/kevinlinxp/sublimeSnippetsSupport/SublimeSnippetsSupportSettings.kt). Would you please let me know the default sublime package folder in your OS? Then I can fix it and release a new version.

Regards,
Kevin

**Comment by kevinlinxp** [comment]
Published v1.2.6 for the fix.
